storages/backends/ftp.py

```diff
@@ -165,6 +166,20 @@ def _get_dir_details(self, path):
         except ftplib.all_errors:
             raise FTPStorageException('Error getting listing for %s' % path)
 
+    def modified_time(self, name):
+        self._start_connection()
+        resp = self._connection.sendcmd('MDTM ' + name)
+        if resp[:3] == '213':
+            s = resp[3:].strip()
+            # workaround for broken FTP servers returning responses
+            # starting with e.g. 1904... instead of 2004...
+            if len(s) == 15 and s[:2] == '19':
+                s = repr(1900 + int(s[2:5])) + s[5:]
```

Comment:
Yeah, **str** would probably be a better option. I copied this from some path to python's ftplib which was never accepted.